openmetadata-ui/src/main/resources/ui/src/components/Users/Users.component.tsx

```diff
@@ -778,7 +784,7 @@ const Users = ({
         <div className="tw--mt-4 tw-px-1.5 tw-flex tw-justify-between">
           <div className="tw-relative">
             <AntDButton
```

Comment:
let's name this as a Button, and if we are using any custom button component, this is a good chance to replace that with antd button